.github/workflows/rundoc.yml

```diff
@@ -0,0 +1,41 @@
+name: run-rundoc
+run-name: running rundoc on tutorials
+on:
+  schedule:
+    # This schedule triggers the workflow at 00:00 every Monday
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+jobs:
+  rundoc-ubuntu:
+    runs-on: pub-hk-ubuntu-24.04-ip
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3.2'
+          bundler-cache: true # do run 'bundle install' and cache
+          working-directory: .rundoc-workspace
+      - uses: buildpacks/github-actions/setup-pack@v5.7.1
+      # [CONFIG] here for the path of what we want to generate
+      - run: bundle exec rundoc ../docs/src/ruby/ruby_tutorial.md --on-success-dir ../docs/ruby --on-failure-dir ../docs/fail --force
+        working-directory: .rundoc-workspace
+        continue-on-error: true
+      - run: rm -rf ruby-getting-started
+        working-directory: docs/ruby
+        continue-on-error: true
+      # [CONFIG] then here move and rename README.md to the appropriate place
+      - run: echo "NOW=$(date +'rundoc-linux/%s')" >> $GITHUB_ENV
+      - run: echo "HUMAN_DATE=$(date +'update tutorials - %Y %b %d')" >> $GITHUB_ENV
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          branch: ${{ env.NOW }}
```

Comment:
Using `env.NOW` for the branch means every time the job runs the PR it opens will have a different branch, and we'll end up with multiple PRs if the previous one wasn't merged yet. I think it would be best to use a fixed branch name, which causes the `peter-evans/create-pull-request` action to automatically update any existing PR, should it still be open (this strategy is what we use for our other automations that use `peter-evans/create-pull-request`).

See also:
https://github.com/peter-evans/create-pull-request#action-behaviour

Reply:
@lilacstella, please take this one.